src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseTestService.java

```diff
@@ -1759,9 +1759,9 @@ void configureRepository_testBadRequestError() throws Exception {
         Team team = setupTeamForBadRequestForStartExercise();
 
         // Start participation
-        var exception = assertThrows(VersionControlException.class, () -> participationService.startExercise(exercise, team, false));
+        assertThatExceptionOfType(VersionControlException.class).isThrownBy(() -> participationService.startExercise(exercise, team, false))
+                .matches(exception -> !exception.getMessage().isEmpty());
         // We cannot compare exception messages because each vcs has their own. Maybe simply checking that the exception is not empty is enough?
```

Comment:
```suggestion
```
Or move up. However, "maybe" comments are usually not really worth keeping

Reply:
here as well